.gitignore

```diff
@@ -1 +1,2 @@
 blogs.yml
+application.sqlite3
```

Comment:
I think we probably want to ignore this for the same reason we ignore our `blogs.yml` file 😁

Reply:
I agree 😁